subsys/llext/llext.c

```diff
@@ -255,6 +290,12 @@ static int llext_map_sections(struct llext_loader *ldr, struct llext *ext)
 		ldr->sect_map[i] = mem_idx;
 	}
 
+	ldr->prog_data_size = high_shdr.sh_size + high_shdr.sh_offset - low_shdr.sh_offset;
+
+	/* No verbatim .rodata, use an automatically selected one */
+	if (!ldr->sects[LLEXT_MEM_RODATA].sh_addr)
```

Comment:
zephyr (and MISRA) wants the brackets for the body here

Reply:
fixed